expression/util.go

```diff
@@ -357,3 +357,81 @@ func extractFiltersFromDNF(ctx sessionctx.Context, dnfFunc *ScalarFunction) ([]E
 	}
 	return extractedExpr, ComposeDNFCondition(ctx, newDNFItems...)
 }
+
+// GetRowLen gets the length if the func is row, returns 1 if not row.
```

Comment:
row(1) will also return 1.

Reply:
So 1 is the length of the row, the first case covers it.